src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background: $color-white;
+  border: 1px solid $color-grey;
```

Comment:
it makes sense to create a variable only when you use some value more than once

Reply:
fix it everywhere

Reply:
removed var $color-grey